programs/protocol_event/src/state/participant.rs

```diff
@@ -0,0 +1,32 @@
+use crate::state::grouping::{MAX_CODE_LENGTH, MAX_NAME_STRING_LENGTH};
+use crate::state::type_size::{
+    vec_size, CHAR_SIZE, DISCRIMINATOR_SIZE, ENUM_SIZE, PUB_KEY_SIZE, U16_SIZE,
+};
+use anchor_lang::prelude::*;
+
+#[account]
+pub struct Participant {
+    pub category: Pubkey,
+    pub participant_type: ParticipantType,
+
+    pub name: String,
+    pub code: String,
+
+    pub id: u16,
+    pub payer: Pubkey,
+}
+
+#[derive(AnchorSerialize, AnchorDeserialize, Debug, Clone, PartialEq, Eq)]
+pub enum ParticipantType {
+    Individual,
+    Team,
+}
+
+impl Participant {
+    pub const SIZE: usize = DISCRIMINATOR_SIZE
+        + PUB_KEY_SIZE * 3
```

Comment:
should it be `x2` not `x3`?

Reply:
yeah good spot, there used to be another Pubkey here